spot_utils/perception/capture.py

```diff
@@ -0,0 +1,112 @@
+"""Functions for collecting images from different cameras and camera sources on
+the spot."""
+
+from typing import Any
+
+import cv2
+import numpy as np
+from bosdyn.api.image_pb2 import Image
+from bosdyn.client.frame_helpers import BODY_FRAME_NAME, get_a_tform_b
+from bosdyn.client.image import _depth_image_data_to_numpy, build_image_request
+
+from spot_utils.structures.image import DepthImage, Intrinsics, RGBDImage, RGBImage
+from spot_utils.structures.robot import RobotClient
+
+CAMERAS = ["frontleft", "frontright", "left", "right", "back", "hand"]
+
+SOURCES_DEPTH_FRAME = {
+    cam: (cam + "_depth", cam + "_visual_in_depth_frame") for cam in CAMERAS[:-1]
+}
+SOURCES_DEPTH_FRAME["hand"] = ("hand_depth", "hand_color_in_hand_depth_frame")
+
+SOURCES_RGB_FRAME = {
+    cam: (cam + "_depth_in_visual_frame", cam + "_fisheye_image")
+    for cam in CAMERAS[:-1]
+}
+SOURCES_RGB_FRAME["hand"] = ("hand_depth_in_hand_color_frame", "hand_color_image")
+
+
+def get_intrinsics(response: Any):
+    """Get the camera intrinsics from an image response."""
+    intrinsics = response.source.pinhole.intrinsics
+    return Intrinsics(
+        response.source.rows,
+        response.source.cols,
+        intrinsics.focal_length.x,
+        intrinsics.focal_length.y,
+        intrinsics.principal_point.x,
+        intrinsics.principal_point.y,
+    )
+
+
+def capture_rgb(robot_client: RobotClient, source: str):
+    """Capture just the rgb image with a particular camera source."""
+
+    # pylint:disable=no-member
```

Comment:
I think this will disable the check for this entire function, is that the intention? if it's just line then you can put it at the end of the line